Plugins/SwiftProtobufPlugin/plugin.swift

```diff
@@ -88,6 +95,14 @@ struct SwiftProtobufPlugin: BuildToolPlugin {
 
         try validateConfiguration(configuration)
 
+        let importPaths: [Path]
+        if let configuredImportPaths = configuration.importPaths {
+            importPaths = configuredImportPaths.map { Path($0) }
+        } else {
+            // We include the target directory as the default
+            importPaths = [target.directory]
```

Comment:
I think we probably should always include the `target.directory` even when `importPaths` are specified.

Reply:
sounds fair to me